docs/references/sdk/frontend-only.mdx

```diff
@@ -1,7 +1,152 @@
 ---
-description: TODO
+description: A reference for implementing a frontend-only Clerk SDK
 ---
 
 # Frontend-only SDK
 
-TODO
+While [ClerkJS](/docs/references/javascript/overview) can be used in any browser context and framework, realistically users expect to consume its features through the conventions and syntax of their framework of choice. `@clerk/clerk-react` turns ClerkJS into React components, `@clerk/astro` into Astro components, and so on.
+
+In non-browser environments (e.g. iOS) you’ll need to re-implement the [Clerk class](/docs/references/javascript/clerk/clerk) in the SDK’s programming language, interacting with the [FAPI](/docs/references/sdk/terminology).
```

Comment:
It feels a bit weird to tell people that they'll need to reimplement the Clerk class for iOS as an example when we've got the iOS SDK.